thunder/tests/test_dynamo.py

```diff
@@ -1370,7 +1374,9 @@ def foo(x):
     py_files = list(tmp_path.rglob("*.py"))
     assert len(py_files) == 16
 
-    for file in py_files:
+    # Randomly select 2 files to run to save time
+    selected_files = random.sample(py_files, min(2, len(py_files)))
```

Comment:
Also would need to use something like `hypothesis` for random sampling here and elsewhere